openfoodfacts/dataset.py

```diff
@@ -57,7 +73,12 @@ def get_dataset(
 
 
 class ProductDataset:
-    def __init__(self, dataset_type: DatasetType = DatasetType.jsonl, **kwargs):
+    def __init__(
```

Comment:
slight break changing, as the documentation gives an explicit usage example with `ProductDataset("csv")`, which will not work anymore

```
// instead
ProductDataset(dataset_type="csv")

// or
from openfoodfacts.types import Flavor
ProductDataset(flavor=Flavor.off, dataset_type="csv")
```